ibm_cloud_sdk_core/jwt_token_manager.py

```diff
@@ -224,3 +104,9 @@ def _request(self,
             return response.json()
 
         raise ApiException(response.status_code, http_response=response)
+
+    @abstractmethod
+    def request_token(self) -> None:  # pragma: no cover
+        """Should be overridden by child classes.
+        """
+        pass
```

Comment:
Do we need this here, since it is already defined in `TokenManager`?

Reply:
Good catch! I've just removed it.
If you think this PR is okay now, can you please merge it?